utils.js

```diff
@@ -7,7 +7,9 @@ async function getSelfUser() {
     },
   });
   const self_user = await res.json();
-  return self_user;
+  if (res.status === 200) {
+    return self_user;
+  }
```

Comment:
What if i get 400 then nothing is sent back also my page goes blank like this

![image](https://user-images.githubusercontent.com/57758447/214205842-b70dbb31-7e2a-4996-ae76-468f71be11f5.png)

Reply:
done